integration/src/test/java/org/mule/test/routing/AsyncTestCase.java

```diff
@@ -115,14 +119,14 @@ public void withMaxConcurrencyAsyncDispatched() throws Exception {
   }
 
   @Test
-  @Story(BACKPRESSURE)
+  @Stories({@Story(BACKPRESSURE), @Story(ExecutionEngineStory.MAX_CONCURRENCY)})
```

Comment:
Static import.

Reply:
cannot, there's already a constant with that identifier